vagrant/provisioning/roles/arkcase-ldap-authority/tasks/main.yml

```diff
@@ -4,4 +4,4 @@
     microservice:
       name: arkcase-ldap-authority
       version: "{{ arkcase_ldap_authority_version | default(arkcase_version) }}"
-      java_opts: "-Dserver.port=9090 -Dspring.cloud.config.uri=https://{{ config_server_host | default(internal_host) }}/config -Dspring.profiles.include=server,runtime -Djava.net.preferIPv4Stack=true -Djavax.net.ssl.trustStorePassword={{ java_trust_store_pass }} -Djavax.net.ssl.keyStore={{ java_key_store }} -Djavax.net.ssl.trustStore={{ java_trust_store}} -Djavax.net.ssl.keyStorePassword={{ java_key_store_pass }}"
+      java_opts: "-Dserver.port=9090 -Dspring.cloud.config.uri=https://{{ config_server_host | default(internal_host) }}/config -Dspring.profiles.include=server,runtime -Djava.net.preferIPv4Stack=true -Djavax.net.ssl.trustStorePassword={{ java_trust_store_pass }} -Djavax.net.ssl.keyStore={{ java_key_store }} -Djavax.net.ssl.trustStore={{ java_trust_store}} -Djavax.net.ssl.keyStorePassword={{ java_key_store_pass }} -Darkcase.ldap.url={{ ldap_portal_url | default(ldap_url) }} -Darkcase.ldap.base={{ ldap_portal_base | default(ldap_base) }} -Darkcase.ldap.manager-dn={{ ldap_portal_bind_user | default(ldap_bind_user) }} -Darkcase.ldap.manager-password={{ ldap_portal_bind_password | default(ldap_bind_password) }} -Darkcase.ldap.search-properties.user-domain={{ ldap_portal_user_domain | default(ldap_user_domain) }} -Darkcase.ldap.search-properties.user-search-base={{ ldap_portal_user_no_base | default(ldap_user_no_base) }} -Darkcase.ldap.search-properties.group-search-base={{ ldap_portal_group_no_base | default(ldap_group_no_base) }} -Darkcase.ldap.search-properties.user-prefix=portal."
```

Comment:
Right now we use this microservice only for the portal. We will need to change this when we start using it for ArkCase.